lib/async_hooks.js

```diff
@@ -351,6 +353,47 @@ class AsyncLocalStorage {
       return resource[this.kResourceStore];
     }
   }
+
+  bindToTracingChannel(channel, transform = (v) => v) {
+    if (typeof channel === 'string') {
+      channel = tracingChannel(channel);
+    }
+
+    const store = this;
+
+    function start(register) {
+      register(store, transform);
+    }
+
+    channel.subscribe({ start });
+    return () => {
+      channel.unsubscribe({ start });
+    };
+  }
+
+  static runInTracingChannel(channel, data, runner) {
```

Comment:
I assume this needs `thisArg, ...args` to be forwarded to the runner via `traceSync`.

Reply:
just noticed that `AsyncLocalStore#run` has no support to provide `thisArg`...